src/components/document/document.tsx

```diff
@@ -55,17 +55,14 @@ const DownloadButton = ({ onClick }: { onClick: SVGClickHandler }) => {
 const PublishButton = ({ onClick, dataTestName }: { onClick: () => void, dataTestName?: string }) => {
   return (
     <IconButton icon="publish" key="publish" className="action icon-publish" dataTestName={dataTestName}
-                onClickButton={onClick} title="Publish Workspace" />
+                onClickButton={onClick} title="Publish To Workspace" />
   );
 };
 
 const PublishSupportButton = ({ onClick }: { onClick: () => void }) => {
   return (
-    <>
-      <IconButton icon="publish-support" key="support" className="action icon-support"
-                  onClickButton={onClick} title="publish to supports" />
-      <div className="support-badge"/>
-    </>
+    <IconButton icon="publish" key="support" className="action icon-publish"
```

Comment:
A comment to indicate that we're not currently publishing as a support would be useful.